qml/pages/LoginPage.qml

```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import harbour.kaidan 1.0
+
+Page {
+	id: loginPage
+	Column {
+		anchors.fill: parent
+
+		PageHeader {
+			title: qsTr("Login")
+		}
+
+		TextField {
+			id: jidField
+			placeholderText: "user@example.org"
+			label: qsTr("JID")
+			width: parent.width
+		}
+
+		TextField {
+			id: passField
+			placeholderText: qsTr("Password")
+			echoMode: TextInput.Password
+			label: qsTr("Password")
+			width: parent.width
+		}
+
+		//It's possibly would be better to make a Dialog
```

Comment:
"It's possibly better to use a dialog." or "It would possibly be better to use a dialog"